hub-templates/daskhub/values.yaml

```diff
@@ -45,6 +52,7 @@ base-hub:
         # The default worker image matches the singleuser image.
         DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE: '{JUPYTER_IMAGE_SPEC}'
 
+        # FIXME: Only set these if scratchBucket.enabled is true
```

Comment:
Any plans about how to make this happen? Doing stuff with some `extraConfig` or something else?

Reply:
Yeah - that, or something with whatever we decide to do in https://github.com/2i2c-org/pilot-hubs/issues/362.